src/sentry/notifications/manager.py

```diff
@@ -199,9 +180,10 @@ def update_settings(
             if not created and setting.value != value.value:
                 setting.update(value=value.value)
 
-            UserOption.objects.set_value(
-                user, key=key, value=legacy_value, project=project, organization=organization
-            )
+            if target.type == 1:
```

Comment:
what is 1 in this case?

Reply:
`1` means "user". I'll update the PR to use the enum.